DavidAsmCore/OpEmitter.cs

```diff
@@ -12,6 +12,7 @@ public enum Opcode : byte
     {
         Val = 0x40,
         Add = 0x10,
+        Sub = 0x11,
```

Comment:
can you double check this has the right encoding?

Reply:
Yes, I'll check. The binary encoding is "00010001" which is 0x11 in hexadecimal.